packages/transport-test/e2e/tests/bridge.test.ts

```diff
@@ -93,35 +84,36 @@ describe('bridge', () => {
                 type: 'Features',
                 message: {
                     vendor: 'trezor.io',
-                    label: 'TrezorT',
                 },
             },
         });
     });
 
-    test(`call(ChangePin) - send(Cancel) - receive`, async () => {
-        // initiate change pin procedure on device
-        const callResponse = await bridge.call({ session, name: 'ChangePin', data: {} }).promise;
+    test(`call(RebootToBootloader) - send(Cancel) - receive`, async () => {
+        // initiate RebootToBootloader procedure on device (it works regardless device is wiped or not)
+        const callResponse = await bridge.call({ session, name: 'RebootToBootloader', data: {} })
+            .promise;
         expect(callResponse).toMatchObject({
             success: true,
             payload: {
                 type: 'ButtonRequest',
             },
         });
 
-        // cancel change pin procedure
+        // cancel RebootToBootloader procedure
         const sendResponse = await bridge.send({ session, name: 'Cancel', data: {} }).promise;
         expect(sendResponse).toEqual({ success: true, payload: undefined });
 
         // receive response
         const receiveResponse = await bridge.receive({ session }).promise;
+
         expect(receiveResponse).toMatchObject({
             success: true,
             payload: {
                 type: 'Failure',
                 message: {
                     code: 'Failure_ActionCancelled',
-                    message: 'Cancelled',
+                    message: USE_HW ? 'Action cancelled by user' : 'Cancelled',
```

Comment:
this message is related to FW version, `Action cancelled by user'` is only a fallback added by DeviceCommands.

i got `Cancelled` message here while testing.

i guess testing `code: 'Failure_ActionCancelled'` should be enough

Reply:
right. it could be difference between T1 and TT.